.circleci/config.yml

```diff
@@ -46,3 +46,21 @@ jobs:
         at: .
 
     - run: ./architect publish --stable
+
+workflows:
+  version: 2
+  build_e2e:
+    jobs:
+      - build
+
+      - deploy:
+          filters:
+            branches:
+              only: master
```

Comment:
Hey, I missed this but we should require the build step. Can we go with mine? #102